### ISSUE TITLE
Accessibility: role="radio" payment option missing required aria-checked when no selection

### ISSUE THREAD

**Issue by sundy001** [Adyen/adyen-web#3824]
**Describe the bug**
The payment payment option buttons are implemented with `role="radio"` but does not have `aria-checked="false"` when the user hasn’t selected any payment method yet. According to the WAI‑ARIA specification, when using role="radio", the aria-checked attribute is required (not optional).

Spec reference: 
https://www.w3.org/TR/wai-aria-1.2/#radio

**To Reproduce**
1. Go to any payment page which use `Dropin` from `@adyen/adyen-web/auto`.
2. Ensure no payment method is selected
3. Inspect the payment method element(s) that use role="radio" (e.g., in DevTools Elements panel).
4. Observe that `aria-checked` is missing (it should be present and set to "false" for unselected radios).

**Expected behavior**
All elements with `role="radio"` should always include `aria-checked`.

**Screenshots**

<img width="1068" height="436" alt="Image" src="https://github.com/user-attachments/assets/f1970824-20b0-4ba0-bb1d-dd65df627b29" />

**Desktop (please complete the following information):**
 - OS: macOS 26.2
 - Browser chrome 144.0.7559.135 (Official Build) (arm64)
 - @adyen/adyen-web@6.29.0

**Comment by m1aw** [comment]
Hey @sundy001 

This is indeed a bug we will fix it in one of the upcoming release.
